### PR TITLE
Adding Transparencies

### PR DESCRIPTION
**Pull request by tyler1218hatch**
Fixes issue #66.

### CODE REVIEW

File: RaveAddIn/XML/brat.xml [hunk 1]
```diff
@@ -82,16 +82,16 @@
           </Node>
           <Node label="Vegetation">
             <Children>
-              <Node label="Existing Vegetation Type" xpath="Raster[@id='EXVEG']" type="raster" symbology="ExistVeg" />
-              <Node label="Historic Vegetation Type" xpath="Raster[@id='HISTVEG']" type="raster" symbology="HistVeg" />
+              <Node label="Existing Vegetation Type" xpath="Raster[@id='EXVEG']" type="raster" symbology="ExistVeg" transparency="40"/>
+              <Node label="Historic Vegetation Type" xpath="Raster[@id='HISTVEG']" type="raster" symbology="HistVeg" transparency="40"/>
```
Comment (joewheaton) [inline RaveAddIn/XML/brat.xml:86]: These transparency additions are good. Just noting that this business logic needs to be expanded to include all the stuff in the PyBRAT layer packages.  My understanding is @MattReimer is close to a fix on getting attribute tables in. Fix in another branch and pull request.

File: RaveAddIn/XML/vbet.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
           <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_90']" type="vector" symbology="" />
           <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='THRESH_100']" type="vector" symbology="" />
           <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Vector[@id='VBET_NETWORK']" type="vector" symbology="" />
-          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Raster[@id='EVIDENCE']" type="raster" symbology="" />
+          <Node xpathlabel="Name" xpath="Realizations/VBET/Intermediates/Raster[@id='EVIDENCE']" type="raster" symbology="" transparency="40"/>
```
Comment (joewheaton) [inline RaveAddIn/XML/vbet.xml:38]: Make sure these VBET changes don't collide with @Cashe93 changes to VBET. Some overlap.

File: RaveAddIn/XML/brat.xml (hunk 1 above)
Comment (MattReimer) [reply RaveAddIn/XML/brat.xml:86]: Yup. We have them now. Will need to propagate but we have the technology

Comment (philipbaileynar) [reply RaveAddIn/XML/brat.xml:86]: Agreed. This looks good.

File: RaveAddIn/XML/vbet.xml (hunk 2 above)
Comment (tyler1218hatch) [reply RaveAddIn/XML/vbet.xml]: @Cashe93 I think we can avoid this as long as you add transparencies when you merge your branch from issue #68 .

**Comment by philipbaileynar** [comment]
@tyler1218hatch I'm going to rebase this today and out out a rave release. Ok?

**Pull request merged**